### ISSUE TITLE
How to connect to external DB?

### ISSUE THREAD

**Issue by peterjc85** [goharbor/harbor#13467]
I built an external DB.
k8s is not used.
I want to connect to the harbor server after building an external DB.
Installation / DB table creation has been completed.

When I run harbor, I get an error in the UI.
"Core service is not available."

What should I do?

**Comment by reasonerjt** [comment]
You need to configure in the harbor.yml to connect to an external DB.
Please be reminded that only pgSQL is supported.

**Comment by ywk253100** [comment]
Not sure it's related with the external database or not. You should to check the log first to find out the root cause of the error

**Comment by peterjc85** [comment]
> Not sure it's related with the external database or not. You should to check the log first to find out the root cause of the error

harbor was built,
I installed postgresql on another VM.
And in postgresql, 4 tables and user are created: registry, clair, notaryserver, notarysigner.
And after setting Listen to "*", I started harbor.
There is no failure log.
However, I can't even log in from the Login Page.
Looking at the DB, there is a table created, but the values ​​do not exist.
Do I have to do something else?

And I configured L4-harbor01, harbor02, and now each of them uses basic DB. The users harbor01, harbor02 are adding manually because they are out of sync.
If only 1EA of external DB is configured and both harbor01 and harbor02 connect only DB1, does it work properly?
Of course, DB also plans to configure HA.
In this case, if harbor01 and harbor02 look at the VIP of the DB, and the DB configures HA (Active / Standby), is it HA configuration?
The k8s are not building.

I want to use harbor with HA configuration.
